src/main/java/com/hackathonorganizer/hackathonwriteservice/team/exception/handler/MethodArgumentNotValidExceptionHandler.java

```diff
@@ -0,0 +1,23 @@
+package com.hackathonorganizer.hackathonwriteservice.team.exception.handler;
+
+import com.hackathonorganizer.hackathonwriteservice.team.exception.ErrorMessage;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import java.time.LocalDateTime;
+
+@RestControllerAdvice
+public class MethodArgumentNotValidExceptionHandler {
+
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    public ErrorMessage handle(MethodArgumentNotValidException e) {
+        return new ErrorMessage(
+                HttpStatus.BAD_REQUEST.toString(),
```

Comment:
@ResponseStatus zwraca juz ten status w odpowiedzi, po co dodatkowo go dodawac w ErrorMessage

Reply:
Pole jaki blad się pojawił powinno być moim zdaniem zwrócone https://betterprogramming.pub/tips-on-rest-api-error-response-structure-aebe726e7f94 jak otworzysz stronę w przeglądarce to nie wiadomo jaki jest http status dopóki nie wejdziesz w narzędzia programistyczne. Kwestia wypracowania wspólnej wersji jak ten error message ma wyglądać